kong/plugins/ai-proxy/handler.lua

```diff
@@ -0,0 +1,133 @@
+local _M = {}
+
+-- imports
+local ai_shared = require("kong.plugins.ai-proxy.drivers.shared")
+local fmt = string.format
+local kong_utils = require("kong.tools.utils")
+local kong_meta = require "kong.meta"
+--
+
+_M.PRIORITY = 770
+_M.VERSION = kong_meta.version
+
+local INTERNAL_SERVER_ERROR_TEMPLATE = '{"error":{"message":"%s"}}'
```

Comment:
Looks a tad risky of generating invalid JSON if the input contains `"`. I think it's worth the extra memory/cycles to do `cjson.encode({ error = { message = ... } })` instead of relying on `string.format`.